lib/ansible/utils/module_docs_fragments/k8s_resource_options.py

```diff
@@ -38,4 +38,5 @@ class ModuleDocFragment(object):
     - Reads from the local file system. To read from the Ansible controller's file system, use the file lookup
       plugin or template lookup plugin, combined with the from_yaml filter, and pass the result to
       I(resource_definition). See Examples below.
+    - To read Ansible Vault-encrypted files, use the file lookup plugin. See Examples below. 
```

Comment:
I would rephrase this slightly to be: 
```suggestion
    - To read Ansible Vault-encrypted files, use the I(resource_definition) parameter instead along with the C(file) lookup plugin. See Examples below. 
```